src/commands/workflow/run.ts

```diff
@@ -0,0 +1,47 @@
+import { Flags } from "@oclif/core";
+
+import * as ApiV1 from "@/lib/api-v1";
+import BaseCommand from "@/lib/base-command";
+import { ApiError } from "@/lib/helpers/error";
+import { commaSeparatedArray, jsonStr } from "@/lib/helpers/flag";
+import {
+  formatErrorRespMessage,
+  isSuccessResp,
+  withSpinner,
+} from "@/lib/helpers/request";
+
+export default class WorkflowRun extends BaseCommand {
+  static flags = {
+    environment: Flags.string({ default: "development" }),
+    recipients: commaSeparatedArray({
+      required: true,
+      aliases: ["recipient"],
+      multiple: true,
+    }),
+    actor: Flags.string(),
+    tenant: Flags.string(),
+    data: jsonStr(),
+  };
+
+  static args = [{ name: "workflowKey", required: true }];
+
+  async run(): Promise<void> {
+    const { args, flags } = this.props;
+
+    const resp = await withSpinner<ApiV1.RehearseWorkflowResp>(
+      () => {
+        return this.apiV1.runWorkflow(this.props);
+      },
+      { action: "‣ Running" },
+    );
+
+    if (!isSuccessResp(resp)) {
+      const message = formatErrorRespMessage(resp);
+      return this.error(new ApiError(message));
+    }
+
+    this.log(
+      `‣ Successfully ran \`${args.workflowKey}\` workflow in \`${flags.environment}\` environment. Run id: ${resp.data.workflow_run_id}.`,
+    );
```

Comment:
```suggestion
    this.log(
      `‣ Successfully ran \`${args.workflowKey}\` workflow in \`${flags.environment}\` environment`,
    );
    this.log(indentString(`Workflow run id: ${resp.data.workflow_run_id}`), 4));
```
Formatting nit, feel free to ignore.